### PR TITLE
remove last sonata admin stuff

### LINKED ISSUE #315: Remove forgotten sonata stuff
at least i forgot the phpcr-admin-bundle in the dev dependencies.

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
fix #315

I did forget some stuff.

### CODE REVIEW

File: Tests/Resources/app/AppKernel.php [hunk 1]
```diff
@@ -20,8 +20,7 @@ public function configure()
 
         if ('phpcr' === $this->environment) {
             $this->requireBundleSets(array(
-                'phpcr_odm',
-                'sonata_admin_phpcr',
```
Comment (ElectricMaxxx) [inline Tests/Resources/app/AppKernel.php]: @wouterj i did so. I removed the complete bundle set in this PR

Comment (ElectricMaxxx) [inline Tests/Resources/app/AppKernel.php]: Btw: as wee need the set in sonata-admin-integration-bundle i would keep it in testing component, or do you think "it is just one usage"?

**Comment by ElectricMaxxx** [comment]
On the failing test i don't get `2.0@dev` of `core-bundle`anymore, so the process part of its extension still tries to add `sonata_admin_extension` as configuration.

**Comment by wouterj** [comment]
> On the failing test i don't get 2.0@dev of core-bundleanymore, so the process part of its extension still tries to add sonata_admin_extension as configuration.

Let's drop CmfCoreBundle 1.* support in `composer.json`

**Comment by dbu** [comment]
thank you!

**Pull request merged**